lib/pure/httpclient.nim

```diff
@@ -228,7 +228,7 @@ proc lastModified*(response: Response | AsyncResponse): DateTime =
   ## Raises a ``ValueError`` if the parsing fails or the value is not a correctly
   ## formatted time.
   var lastModifiedHeader = response.headers.getOrDefault("last-modified")
-  result = parse(lastModifiedHeader, "dd, dd MMM yyyy HH:mm:ss Z")
+  result = parse(lastModifiedHeader, "ddd, dd MMM yyyy HH:mm:ss 'GMT'", utc())
```

Comment:
You're adding a default value here, right? That makes the above doc comment outdated as far as I can see.

Reply:
`utc()` is the timezone, as we read a GMT time

Reply:
Btw, why do we use `GMT` and not `UTC`?

Reply:
`GMT` is just a static part of the format as specified in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified.